### PR TITLE
Update versions and allow builds from unsigned deb repos

### PR DESCRIPTION
**Pull request by theduderog**
@arrawatia @samjhecht @jonchiu 

This keeps things the same by default but if you want to build with an unsigned deb repo, you can override like this:

`make CONFLUENT_DEB_REPO=http://foobar APT_ALLOW_UNAUTHENTICATED=true build-debian`

### CODE REVIEW

File: debian/base/Dockerfile [hunk 1]
```diff
@@ -69,7 +72,7 @@ RUN echo "===> Updating debian ....." \
     && apt-get -y install zulu-${ZULU_OPENJDK_VERSION} \
     && rm -rf /var/lib/apt/lists/* \
     && echo "===> Adding confluent repository...${CONFLUENT_DEB_REPO}/deb/${CONFLUENT_MAJOR_VERSION}.${CONFLUENT_MINOR_VERSION}" \
-    && curl -L ${CONFLUENT_DEB_REPO}/deb/${CONFLUENT_MAJOR_VERSION}.${CONFLUENT_MINOR_VERSION}/archive.key | apt-key add - \
+    && if [ "x$APT_ALLOW_UNAUTHENTICATED" = "xtrue" ]; then echo "APT::Get::AllowUnauthenticated \"true\";" > /etc/apt/apt.conf.d/allow_unauthenticated; else curl -L ${CONFLUENT_DEB_REPO}/deb/${CONFLUENT_MAJOR_VERSION}.${CONFLUENT_MINOR_VERSION}/archive.key | apt-key add - ; fi \
```
Comment (arrawatia) [inline debian/base/Dockerfile]: Is this for repos without a signing key ?

File: debian/base/Dockerfile [hunk 2]
```diff
@@ -57,7 +60,7 @@ RUN echo "===> Updating debian ....." \
                 netcat \
                 python=${PYTHON_VERSION} \
```
Comment (arrawatia) [inline debian/base/Dockerfile]: Should we add support for https://... repos too ? The staging repos are https://...

https://github.com/confluentinc/cp-docker-images/commit/ffdf60c6e7cf5232a822347b351b194fd4a0a9af#diff-8a469355c509e07885a8934541d1dc03L60

File: debian/base/Dockerfile (hunk 1 above)
Comment (theduderog) [inline debian/base/Dockerfile]: Yes

File: debian/base/Dockerfile (hunk 2 above)
Comment (theduderog) [inline debian/base/Dockerfile]: Sure.  Will add.

**Comment by theduderog** [comment]
@arrawatia i addressed the HTTPS support.  any other feedback?

**Comment by jonchiu** [comment]
I tried with https. Lgtm 






On Wed, Jan 25, 2017 at 2:44 PM -0800, "Roger Hoover" <notifications@github.com> wrote:












@arrawatia i addressed the HTTPS support.  any other feedback?

**Pull request merged**